iscsi/iscsi.go

```diff
@@ -45,6 +46,13 @@ type TargetInfo struct {
 	Port   string `json:"port"`
 }
 
+type HCTL struct {
+	HBA     int
+	Channel int
+	Target  int
+	LUN     int
```

Comment:
Are these properties written to a JSON file? If so, please add 'json:"name"' annotations`

Reply:
They are not, this struct matches one added upstream since forking for ease of merging if we do so at some point.

Reply:
Thanks for the clarification.